system-int-test/test/cmr/system_int_test/header_test.clj

```diff
@@ -67,3 +69,16 @@
                               {:headers {:cmr-request-id cmr-request-id}
                                :throw-exceptions false})]
       (is (cmr-request-id-in-header? headers cmr-request-id)))))
+
+(deftest cors-headers
+  (testing "allowed headers in options search request"
+    (is (= (get-in (client/options (url/search-url :collection))
+                   [:headers routes/CORS_CUSTOM_ALLOWED_HEADER])
```

Comment:
We generally try to write our tests without referencing the source code vars so that it's clear what's being tested, and if there is a change to the source we know we did not break anything. 

For example this test would pass as written even without adding in the CMR-Scroll-Id header.

So we'd probably turn this into 4 assertions - one for each of the allowed headers. Something along the lines of:
```
  (let [allowed-headers (-> (client/options (url/search-url :collection))
                            (get-in [:headers "Access-Control-Allow-Headers"])
                            (string/split #", "))]
    (is (some #{"Echo-Token"} allowed-headers))
    (is (some #{"Client-Id"} allowed-headers))
    (is (some #{"CMR-Request-Id"} allowed-headers))
    (is (some #{"CMR-Scroll-Id"} allowed-headers)))
```
Note that you'll need to require `[clojure.string :as string]`